tnb/base_client.py

```diff
@@ -35,14 +35,14 @@ def send_request_from_url(self, method, url, **kwargs):
 
         return response.json()
 
-    def fetch(self, resource, **kwargs):
+    def fetch(self, resource: str, **kwargs: object) -> object:
         """
         GET a `resource` from a Node
         Return response as Python object
         """
         return self.send_request("GET", resource, **kwargs)
 
-    def fetch_multiple_page(self, resource, **kwargs):
+    def fetch_multiple_page(self, resource: str, **kwargs) -> object:
```

Comment:
The parameter `kwargs` is not being used. I wonder if we can remove this parameter. (Maybe a new issue for this)

This method returns a `list`.

**Note**: The method documentation is also incorrect. 
```suggestion
    def fetch_multiple_page(self, resource: str, **kwargs) -> list:
```